src/services/ParticipantProjectSpeciesService.ts

```diff
@@ -190,14 +192,37 @@ const getProjectsForSpecies = async (speciesId: number, organizationId: number):
   const apiSearchResults = (await SearchService.search(searchParams)) as SpeciesProjectsSearchResult[];
   // it will always be one result since we are searching by species id
   const firstApiResult = apiSearchResults[0];
-  const projects = firstApiResult?.participantProjectSpecies.map((pps) => {
-    return {
-      submissionStatus: pps.submissionStatus,
-      projectName: pps.project.name,
-      projectId: pps.project.id,
-      id: pps.id,
-    } as SpeciesProjectsResult;
-  });
+  const projects = await Promise.all(
+    firstApiResult?.participantProjectSpecies.map(async (pps) => {
```

Comment:
I think this can all be done in a single request to the search API. I will put up a PR

Reply:
@constanzauanini let me know what you think https://github.com/terraware/terraware-web/pull/2638